### PR TITLE
colorable on windows

### PR DESCRIPTION
**Pull request by mattn**
Note that you need to bundle latest go-colorable.

### CODE REVIEW

**Comment by maruel** [comment]
I'm not fond of Godeps/_workspace/src/github.com/mgutz/ansi/cmd/ansi-mgutz/ansi-mgutz.exe. Can this file be deleted first so it doesn't get pulled into the history?

**Comment by mattn** [comment]
I'll rebase and remove exe file in later. Sorry.

**Comment by mattn** [comment]
done. Do you want to remain main.go in the root?

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,10 +1,21 @@
 {
 	"ImportPath": "github.com/maruel/panicparse",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "devel +adf4c38 Thu Nov 26 17:42:14 2015 +0000",
```
Comment (maruel) [inline Godeps/Godeps.json]: I'd prefer it to be a real go toolset version, e.g. 1.5.1. But I don't know why this line is embedded at all, this has no value.

**Comment by maruel** [comment]
And yes, I prefer main.go to be left there. In general, I prefer each pull request to be focused on one fix.

File: Godeps/Godeps.json (hunk 1 above)
Comment (mattn) [inline Godeps/Godeps.json]: ah sorry. I always use tip.

On 11/29/15, M-A <notifications@github.com> wrote:
>> @@ -1,10 +1,21 @@
>>  {
>>  	"ImportPath": "github.com/maruel/panicparse",
>> -	"GoVersion": "go1.4.2",
>> +	"GoVersion": "devel +adf4c38 Thu Nov 26 17:42:14 2015 +0000",
>
> I'd prefer it to be a real go toolset version, e.g. 1.5.1. But I don't know
> why this line is embedded at all, this has no value.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/maruel/panicparse/pull/13/files#r46087035


-- 
- Yasuhiro Matsumoto

Comment (mattn) [inline Godeps/Godeps.json]: will fix in later

On 11/29/15, M-A <notifications@github.com> wrote:
>> @@ -1,10 +1,21 @@
>>  {
>>  	"ImportPath": "github.com/maruel/panicparse",
>> -	"GoVersion": "go1.4.2",
>> +	"GoVersion": "devel +adf4c38 Thu Nov 26 17:42:14 2015 +0000",
>
> I'd prefer it to be a real go toolset version, e.g. 1.5.1. But I don't know
> why this line is embedded at all, this has no value.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/maruel/panicparse/pull/13/files#r46087035


-- 
- Yasuhiro Matsumoto

**Comment by mattn** [comment]
done

File: internal/main.go [hunk 2]
```diff
@@ -119,7 +119,7 @@ func IsTerminal(out io.Writer) bool {
 	if !ok {
```
Comment (maruel) [inline internal/main.go]: This function is not used anymore, please remove. Then the PR is good to go. Thanks for your patience.

**Comment by mattn** [comment]
removed IsTerminal.

**Comment by maruel** [comment]
Thanks for the change, it's appreciated!

**Pull request merged**